### ISSUE TITLE
console.log() not supported in all browsers

### ISSUE THREAD

**Issue by mish15** [sajari/sajari-examples#1]
Should wrap this with a check if we want to keep it

**Comment by jonathaningram** [comment]
Yeah could do. Depends how much you want to support old browsers (angular
may even be polyfilling it in this example).
